### DISCUSSION TITLE
question about control point coordinate

### DISCUSSION THREAD

**Discussion by kuailefeifei** [openTCS/opentcs#176] [Q&A]
when i draw a 2-bezier path in model editor, i am confused with the control point coordinate. For example, i put a control point in position (40000, 5000), however, in the properties the control point coordinate is (800, -100). Has someone any ideas about this?
![捕获](https://github.com/user-attachments/assets/d2586cce-7307-4fed-b632-e012c508dc76)

**Comment by martingr** [comment]
This has already been answered here: https://github.com/orgs/openTCS/discussions/13

**Reply by kuailefeifei** [reply]
Thanks for your answer, i got it.
